### PR TITLE
Transaction Reference Content Type

### PR DESCRIPTION
**Pull request by lourou**
https://github.com/orgs/xmtp/discussions/37

### CODE REVIEW

File: packages/content-type-transaction-reference/CHANGELOG.md
Comment (rygine) [inline packages/content-type-transaction-reference/CHANGELOG.md:1]: this file should be auto-generated by changesets upon first publish

File: packages/content-type-transaction-reference/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 XMTP developers (xmtp.org)
```
Comment (rygine) [inline packages/content-type-transaction-reference/LICENSE:3]: `2023` => `2024`

File: packages/content-type-transaction-reference/CHANGELOG.md
Comment (lourou) [reply packages/content-type-transaction-reference/CHANGELOG.md:1]: ok, deleting the file

File: packages/content-type-transaction-reference/src/TransactionReference.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  fallback(content: TransactionReference): string | undefined {
+    if (content.reference) {
+      return `[Crypto transaction] Use a blockchain explorer to learn more using the transaction hash: ${content.reference}`;
+    } else {
+      return `Crypto transaction`;
+    }
+  }
+}
```
Comment (rygine) [inline packages/content-type-transaction-reference/src/TransactionReference.ts:66]: please add empty line at the end

**Comment by nplasterer** [comment]
Looks like some legit CI failures but otherwise looks good. 
Just want to note that this Closes https://github.com/xmtp/xmtp-js-content-types/issues/16

**Review by rygine — APPROVED** [review]

**Pull request merged**